libraries/AC_PID/AC_PID.cpp

```diff
@@ -262,8 +375,12 @@ float AC_PID::get_d() const
 
 float AC_PID::get_ff()
 {
-    _pid_info.FF = _target * _kff;
-    return _target * _kff;
+    _pid_info.FF = _target * _kff
+#if AC_PID_ADVANCED_ENABLED
+        + _target_derivative * _kdff
```

Comment:
Should we be logging this separately?

Reply:
Yeah seems like a good idea

Reply:
Actually we don't log the derivative so that would be inconsistent with that. Happy to go either way.